pandas/io/formats/format.py

```diff
@@ -226,7 +226,79 @@ def to_string(self) -> str:
         return str("\n".join(result))
 
 
-class SeriesFormatter:
+class TableFormatter:
```

Comment:
is there a reason this is moved up from below?

Reply:
Yes - the SeriesFormatter now inherits from TableFormatter, meaning it needs to be declared first.